src/components/PrometheusGraph/PrometheusGraph.js

```diff
@@ -213,6 +213,8 @@ class PrometheusGraph extends React.PureComponent {
       hoverY: null,
       chartWidth: 0,
       chartHeight: 0,
+      timeScale: {},
+      valueScale: {},
```

Comment:
This should be a function (perhaps a `noop`?), otherwise we get console warnings for type mismatch.